docs/web_reference.rst

```diff
@@ -2762,27 +2762,41 @@ Normalize path middleware
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. function:: normalize_path_middleware(*, \
-                                        append_slash=True, merge_slashes=True)
+                                        append_slash=True, \
+                                        remove_slash=False, \
+                                        merge_slashes=True, \
+                                        redirect_class=HTTPMovedPermanently)
 
-  Middleware factory which produces a middleware that normalizes
-  the path of a request. By normalizing it means:
+   Middleware factory which produces a middleware that normalizes
```

Comment:
In sphinx the normal indentation is 3 spaces.
We use this default everywhere in aiohttp docs.
Why did you change it?

Reply:
The indentation in that section was two, it's now three. I noticed the difference when adding the `.. versionadded::` tag.

Reply:
Got it. Sorry, spaces are hard to count in github review tool

Reply:
No problem, I even had to open it back up in my editor to verify because it did look like four in github. 

Thanks!